WifiART/src/structs/LcrState.java

```diff
@@ -0,0 +1,37 @@
+package structs;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class LcrState {
+
+    private List<AddressElement> addressElements;
+
+    private class AddressElement {
+        private final String addressElementLanguage;
+        private final String addressElementDescription;
+        private final String addressElementName;
+
+        AddressElement(String language, String description, String name) {
+            addressElementLanguage = language;
+            addressElementDescription = description;
+            addressElementName = name;
+        }
+    }
+
+    public LcrState() {
+        addressElements = new ArrayList<>();
+    }
+
+    public void addAddressElement(String language, String description, String name) {
+        addressElements.add(new AddressElement(language, description, name));
+    }
+
+    public void removeAddressElement(int index) {
```

Comment:
Same question as above - how would the caller know which index to remove?

Reply:
I'll address the question of the "index" here. You're right that the caller shouldn't know the indices of elements in a list; that would violate the MVC principles. I could instead store the BSSIDs in a content-addressable format, like a HashSet, since it seems duplicate BSSIDs shouldn't be allowed. Then the various classes would not need to keep track of indices, but the order in which the user enters BSSIDs would not be preserved.